portal/views/email.py

```diff
@@ -83,6 +84,22 @@ def verify_email(request, token):
 
 def send_new_users_report(request):
     new_profiles_count = User.objects.filter(date_joined__gte=timezone.now() - timedelta(days=7)).count()
+    users_count = User.objects.count()
+    schools = School.objects.all()
+    school_count = schools.count()
+    teacher_count = Teacher.objects.count()
+    student_count = Student.objects.count()
+    countries = Counter()
+    for school in schools:
+        countries[str(school.country.name)] += 1
+    countries_list = "\n".join('{} {}'.format(k, v) for k, v in countries.items())
```

Comment:
Is this list ordered in any special way.
would be good to have most popular to least popular?
or maybe even just alphabetical.